### DISCUSSION TITLE
unable to install podman 4.6.0

### DISCUSSION THREAD

**Discussion by KamaljeetGulati** [podman-desktop/podman-desktop#4140] [Q&A]
How WSL can install manually on Windows 10 before installing Podman 4.6.0 and which distribution podman used?
Actually I have created package via PSADT script to install podman on user's system via software center and after installing podman 4.6.0 EXE it is asking for restart and after restart when I ran the command :
Execute-Process -Path "C:\Program Files\RedHat\Podman\podman.exe" -Parameters "machine init" -WindowStyle 'Hidden' -Wait -IgnoreExitCodes "125"
it is giving me popup that WSL is not installed and when click OK it is installing and forcedly restart my system but after restart podman machine is not creating.

**Comment by benoitf** [comment]
I think you can follow https://podman-desktop.io/docs/onboarding/containers/installing-podman-with-podman-desktop-on-windows#procedure

`wsl --install --no-distribution`

so no distribution

then you need to do the 'podman machine init' step (on the dashboard you should see a 'Init and Start' button
